src/components/SequenceCard/SequenceCard.js

```diff
@@ -52,8 +64,13 @@ const Card = styled.section`
   border: 1px solid grey;
   margin: 15px 0;
   padding: 0 10px;
+  ${(props) => !props.isActive && 'background-color : #E7C5CA; color: grey;'}
```

Comment:
Wirklich nur eine Winzigkeit: Ich würde bei der Farbbenennung versuchen einen Stil beizubehalten. Entweder HEX, wörtlich oder RGB.

Reply:
yes! 
the design right now is non-existent ;-) the colors are simple template colors. i will use global variable in the project  later. thx for the note.